src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -93,9 +93,14 @@ const UserGeneralSettings = () => {
   );
 
   const UserGeneralSettingsSchema = Yup.object().shape({
-    email: Yup.string()
-      .email(intl.formatMessage(messages.validationemailformat))
-      .required(intl.formatMessage(messages.validationemailrequired)),
+    email:
+      user?.id === 1
+        ? Yup.string()
+            .email(intl.formatMessage(messages.validationemailformat))
+            .required(intl.formatMessage(messages.validationemailrequired))
```

Comment:
I'd recommend swapping these two as we'd like to show the required message when the field is empty. Currently, the email validation will be displayed when the field is empty